### ISSUE TITLE
API Docs - Disable online swagger validator

### ISSUE THREAD

**Issue by Keyhaku** [digitalocean/netbox#2064]
### Issue type
[X] Feature request 
[ ] Bug report      
[ ] Documentation   

### Environment
* Python version:  3.6.5
* NetBox version:  2.3.3

### Description
Hello,

Our instance of netbox is restricted to our internal network only, the problem is that when someone go to the API doc view (/api/docs/), the online swagger validator is triggered, and it delays the whole display of API endpoints to finally display a nice "error badge".

I don't get the point of this online validator on production environnement ...

Thank you

**Comment by Keyhaku** [comment]
It seems there is no way to customize Swagger settings, since you enforce configuration
https://github.com/digitalocean/netbox/blob/develop/netbox/netbox/settings.py#L250

**Comment by bartdebruijn** [comment]
I have the same issue, it would be nice if we could override the online validator setting from the main config file.

**Comment by jeremystretch** [comment]
What is the exact change being proposed? How would you modify the configuration?

**Comment by bartdebruijn** [comment]
@jeremystretch well according to https://github.com/swagger-api/swagger-ui/blob/master/docs/usage/configuration.md you can pass the following to Swagger-UI:


validatorUrl | String="https://online.swagger.io/validator" OR null.  By default, Swagger-UI attempts to validate specs against swagger.io's  online validator. You can use this parameter to set a different  validator URL, for example for locally deployed validators (Validator Badge). Setting it to null will disable validation.
-- | --

Not sure on how to implement that into Netbox. Perhaps if we can set that from configuration.py as an exported variable or something?

**Comment by rijalati** [comment]
So this was an issue for me too, but you can set the validatorUrl via `SWAGGER_SETTINGS` in `settings.py`, here's what mine looks like:

```python
SWAGGER_SETTINGS = {
    'DEFAULT_FIELD_INSPECTORS': [
        'utilities.custom_inspectors.NullableBooleanFieldInspector',
        'utilities.custom_inspectors.CustomChoiceFieldInspector',
        'drf_yasg.inspectors.CamelCaseJSONFilter',
        'drf_yasg.inspectors.ReferencingSerializerInspector',
        'drf_yasg.inspectors.RelatedFieldInspector',
        'drf_yasg.inspectors.ChoiceFieldInspector',
        'drf_yasg.inspectors.FileFieldInspector',
        'drf_yasg.inspectors.DictFieldInspector',
        'drf_yasg.inspectors.SimpleFieldInspector',
        'drf_yasg.inspectors.StringDefaultFieldInspector',
    ],
    'DEFAULT_FILTER_INSPECTORS': [
        'utilities.custom_inspectors.IdInFilterInspector',
        'drf_yasg.inspectors.CoreAPICompatInspector',
    ],
    'DEFAULT_PAGINATOR_INSPECTORS': [
        'utilities.custom_inspectors.NullablePaginatorInspector',
        'drf_yasg.inspectors.DjangoRestResponsePagination',
        'drf_yasg.inspectors.CoreAPICompatInspector',
    ],
    'VALIDATOR_URL': None
}
```
which sets the UI's `validatorUrl` to `Null`. Here's what the `swagger-settings` json object looks like when I view the page source:
```html
<script id="swagger-settings" type="application/json">{"docExpansion": "list", "deepLinking": false, "showExtensions": true, "defaultModelRendering": "model", "defaultModelExpandDepth": 3, "defaultModelsExpandDepth": 3, "showCommonExtensions": true, "supportedSubmitMethods": ["get", "put", "post", "delete", "options", "head", "patch", "trace"], "validatorUrl": null}</script>
```

**Comment by jeremystretch** [comment]
Fixed in ef61c70a9dc36969678691afed6b9dbedb33963c
